config/vufind/searchspecs.yaml

```diff
@@ -186,6 +186,13 @@ ISN:
     - issn
   DismaxHandler: edismax
 
+BasicClassification:
```

Comment:
We won't want to include this definition in the actual searchspecs.yaml when we merge everything -- can we just move the relevant information to the comments at the top? (We don't need to make this change right away -- might as well wait until the end so we document the final form everything takes, in case adjustments are made along the way). Just leaving this comment as a reminder to take care of that eventually!